bin/lb-ng

```diff
@@ -31,6 +31,10 @@ if (app.booting) {
 }
 
 function runGenerator() {
+  // setting stdout to blocking so process.nextTick does not get interupted
+  // while stdout.write isn't finished in child_process on node v6
+  process.stdout._handle.setBlocking(true);
```

Comment:
doesn't work on 0.10 and 0.12, is that OK? As long as you make sure it only occurs for the generator and doesn't effect any other sub-commands, fine by me.